com.unity.multiplayer.mlapi/Documentation~/MigrateFromMLAPI.md

```diff
@@ -0,0 +1,31 @@
+# Migrate from MLAPI
+
+Something something, this package was MLAPI but now it's Netcode for GameObjects bla bla.
+
+## Target Audience
+
+people coming from MLAPI 0.1.0 package released in March
+
+## What's Changed?
+
+asmdef, namespaces and package.json
+
+## Auto API Updater
+
+on 2020.3: non-asmdef'd scripts
+
+on 2021.1+: all scripts
+
+## Manual Update
+
+asmdef refs, namespaces
+
+## Known Issues
```

Comment:
might not be necessary due to Manual Update section and decision on API Upgrader